integration_test/workflow_tests/test_packages/ingest/test_multuple_granules_happy_path.py

```diff
@@ -0,0 +1,197 @@
+                    "excludedFileExtensions": "{$.meta.collection.meta.orca.excludedFileExtensions}",
+                    "s3MultipartChunksizeMb": "{$.meta.collection.meta.s3MultipartChunksizeMb}",
+                    "providerId": "{$.meta.provider.id}",
+                    "providerName": "{$.meta.provider.name}",
+                    "executionId": "{$.cumulus_meta.execution_name}",
+                    "collectionShortname": "{$.meta.collection.name}",
+                    "collectionVersion": "{$.meta.collection.version}",
+                    "defaultBucketOverride": "{$.meta.collection.meta.orca.defaultBucketOverride}",
+                    "defaultStorageClassOverride": "{$.meta.collection.meta.orca.defaultStorageClassOverride}"
+                },
+                "exception": "None"
+                }
+
+            execution_info = boto3.client("stepfunctions").start_execution(
+                stateMachineArn=helpers.orca_copy_to_archive_step_function_arn,
+                input=json.dumps(copy_to_archive_input, indent=4),
+            )
+
+            step_function_results = helpers.get_state_machine_execution_results(
+                execution_info["executionArn"]
+            )
+
+            self.assertEqual(
+                "SUCCEEDED",
+                step_function_results["status"],
+                "Error occurred while starting step function.",
+            )
+            self.assertEqual(
+                expected_output,
+                json.loads(step_function_results["output"]),
+                "Expected step function output not returned.",
+            )
+            # verify that the objects exist in recovery bucket
+            try:
+                for keys in [key_name_1, key_name_2]:
+                    output = boto3.client("s3").head_object(Bucket=recovery_bucket_name, Key=keys)
+                    self.assertEqual(
+                        200,
+                        output["ResponseMetadata"]["HTTPStatusCode"],
+                        f"object {keys} does not exist in the {recovery_bucket_name}",
+                    )            
+            except Exception as ex:
+                return  ex
+
+            catalog_output = helpers.post_to_api(
+                my_session,
+                helpers.api_url + "/catalog/reconcile/",
+                data=json.dumps(
+                    {
+                        "pageIndex": 0,
+                        "providerId": [provider_id],
+                        "granuleId": [granule_id_1, granule_id_2],
+                        "endTimestamp": int((time.time() + 5) * 1000),
+                    }
+                ),
+                headers={"Host": helpers.aws_api_name},
+            )
+            self.assertEqual(
+                200, catalog_output.status_code, "Error occurred while contacting API."
+            )
+            self.assertEqual(
+                {"granules": [granule_id_1, granule_id_2], "anotherPage": False},
```

Comment:
`catalog_output` contains much more information about the granule than just the id. This check will always fail.